docker-compose.yml

```diff
@@ -54,6 +54,7 @@ services:
     volumes:
       - ./db_schema/schema.sql:/docker-entrypoint-initdb.d/schema.sql
       - ./db_schema/metabase.sql:/docker-entrypoint-initdb.d/metabase.sql
```

Comment:
Maybe we should rename this file to make it clearer that this is our custom config.
E.g. `metabase_custom.sql` or `metabase_admin.sql`.

Reply:
changed it to metabase_user